library_nativeiofs.js

```diff
@@ -40,6 +40,10 @@ mergeInto(LibraryManager.library, {
       return parts.join('_');
     },
 
+    encodedPath: function(node) {
```

Comment:
For consistency, could you also use this function in the other place this two calls are directly chained? E.g.:

https://github.com/fivedots/nativeio-emscripten-fs/blob/4649fedabc7c6d136aaa48cd85321071850ab4dc/library_nativeiofs.js#L165

https://github.com/fivedots/nativeio-emscripten-fs/blob/4649fedabc7c6d136aaa48cd85321071850ab4dc/library_nativeiofs.js#L284

Reply:
I'm not convinced it helps in this really helps to make the code more readable, as the path variable is being used in the surrounding code everywhere else.